scripts/ci-entrypoint.sh

```diff
@@ -32,6 +32,9 @@ make --directory="${REPO_ROOT}" "${KUBECTL##*/}" "${HELM##*/}" "${KIND##*/}" "${
 # export the variables so they are available in bash -c wait_for_nodes below
 export KUBECTL
 export HELM
+# initialize variables that are reused across functions
+CIDR0=""
```

Comment:
not sure we need this? we can just use `{CIDR0:-}` where there are places where the variable might not be set yet, but for CIDR0 it seems that it's always set after occurrence and for CIDR1 we can just do `|| CIDR1=""` instead of `|| true` on waht used to be line 137

Reply:
yes, I think we can remove these vars, good catch!